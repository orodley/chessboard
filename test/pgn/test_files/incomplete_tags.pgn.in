1. e4 1-0
