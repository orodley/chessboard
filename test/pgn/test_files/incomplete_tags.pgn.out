[Event "?"]
[Site "?"]
[Date "????.??.??"]
[Round "?"]
[White "?"]
[Black "?"]
[Result "1-0"]

1. e4 1-0
